src/depcache.erl

```diff
@@ -812,3 +840,6 @@ flush_message(Msg) ->
     after 0 ->
             ok
     end.
+
+callback(Type, Name, {M, F, A}) ->
+    spawn(M, F, [{Type, Name} | A]).
```

Comment:
Doing a spawn here to make sure the callback is executed asynchronously. Perhaps this is overkill, and `erlang:apply` is just as fine?